honeybee_radiance_command/pcond.py

```diff
@@ -0,0 +1,83 @@
+"""pcond command."""
+
+from .options.pcond import PcondOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Pcond(Command):
+    """pcond command."""
+
+    __slots__ = ('_input')
```

Comment:
Python is strange! You need a comma - otherwise it will be considered a string not a tuple. It probably still works but it's not good practice.

```suggestion
    __slots__ = ('_input',)
```